client/src/app/routes/stardust/Addr.tsx

```diff
@@ -207,7 +222,36 @@ class Addr extends AsyncComponent<RouteComponentProps<AddrRouteProps>, AddrState
                                                 )}
                                         </div>
                                     </div>
-
+                                    {this.props.match.params.network === 'alphanet-1' && (
```

Comment:
These checks are not needed, as this `stardust/Addr.tsx` will only be rendered if the network has `protocolVersion=stardust`, so it is already checked before.